src/components/AppNavigation/EditCalendarModal.vue

```diff
@@ -172,6 +184,23 @@ export default {
 
 			this.calendarName = this.calendar.displayName
 			this.calendarColor = this.calendar.color
+			if (this.calendar.timezone) {
+				try {
+					const jCal = ICAL.parse(this.calendar.timezone)
+					const iCalComp = new ICAL.Component(jCal)
+					const tzComp = TimezoneComponent.fromICALJs(iCalComp)
+					this.calendarTimezone = tzComp.getFirstComponent('VTIMEZONE').getFirstProperty('TZID').value
```

Comment:
**question (non-blocking):** Do we need to validate the value? Maybe use calendar-js' `TimezoneManager.hasTimezoneForId()` before assigning it?